packages/dx-react-chart-demos/src/demo-sources/chart-basic/bootstrap4/theme.jsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Card } from 'reactstrap';
+import {
+  Chart,
+  PieSeries,
+} from '@devexpress/dx-react-chart-bootstrap4';
+import {
+  schemeCategory10,
+  schemeAccent,
+  schemeDark2,
+  schemePaired,
+  schemePastel1,
+  schemePastel2,
+  schemeSet1,
+  schemeSet2,
+  schemeSet3,
+} from 'd3-scale-chromatic';
+
+import { Scale, Theme } from '@devexpress/dx-react-chart';
+
+const materialUI = ['#2196F3', '#F44336', '#4CAF50', '#FFEB3B', '#E91E63', '#9C27B0'];
```

Comment:
I think that we should use `MaterialDesignColors` instead `MaterialUI` because material-ui is only react library.
Add more colors to that palette from official site https://material.io/design//color/the-color-system.html#tools-for-picking-colors

Reply:
I'll change the name to `MaterialDesignColors`. 
And why are we need to add more color to that palette?

Reply:
Because colors are repeated and it looks bad.

Reply:
But other schemes has not many colors too.